.env.alfajoresstaging

```diff
@@ -77,11 +79,10 @@ FAUCET_CUSD_WEI=60000000000000000000000
 VALIDATORS=10
 PROXIED_VALIDATORS=5
 TX_NODES=5
-STATIC_IPS_FOR_GETH_NODES=true
+STATIC_IPS_FOR_GETH_NODES=false
```

Comment:
Why did we toggle this?

Reply:
Since there aren't any external validators, we don't need to use static ips so other nodes can peer